.github/workflows/kover.yml

```
@@ -0,0 +1,151 @@
+          echo "" >> pr-comment.md
+          echo "## 🔍 커밋된 내용의 테스트 커버리지" >> pr-comment.md
+          echo "" >> pr-comment.md
+          echo "|Files|Diff Coverage (%)|Missing lines|" >> pr-comment.md
+          echo "|-----|-----------------|--------------|" >> pr-comment.md
+
+          # diff-cover 결과를 테이블로 변환
+          tail -n +3 diff-cover.md | while read -r line; do
+            if [[ "$line" == "## Summary"* ]]; then
+              break
+            fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)